magento/magento2
# Cleaner documentation for Travis CI static tests

### Description

This fix applies only to version 2.3.
Travis CI is failing static test for the commited files. The reason is that the comments does not adhere perfectly to the standard, especially on `{@inheritdoc}` usage.
Reference: [inheritdoc documentation](https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.inlineinheritdoc.pkg.html)

### Manual testing scenarios

1. Run static tests on Travis without this commit to see the previous errors. Example: Pull request https://github.com/magento/magento2/pull/18373 Travis CI Build #13951 Job #13951.3
 2. See Travis CI results on this pull request

### Contribution checklist
